wurst/AstarJPS.wurst

```diff
@@ -75,15 +75,17 @@ public class Pathfinder
         return result
 
     function setCond(PathingCondition pred)
-        this.cond = pred
+        if cond != null
```

Comment:
Is there ever a case or even possibility where one would change the condition mid search?

Reply:
Not sure, maybe?